template-pack.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "io.kubeshop.example.templatepack",
+  "name": "Kubeshop Example Template Pack",
+  "author": "Kubeshop",
+  "description": "Example template pack",
+  "templates": [
+    "multiple-forms/template.json",
```

Comment:
these paths seem to be missing the containing folder - i.e. either /templates-helm or /templates-vanilla - should we add them or move the templates out of those folder into the root so the pats in here are correct?